src/vms/pvm/api.ts

```diff
@@ -266,4 +269,29 @@ export class PVMApi extends AvaxApi {
       timestamp: resp.timestamp,
     };
   }
+
+  async getL1Validator(validationID: string): Promise<L1ValidatorDetails> {
+    const resp = await this.callRpc<GetL1ValidatorResponse>('getL1Validator', {
+      validationID,
+    });
+
+    const deactivationOwner = PChainOwner.fromNative(
+      resp.deactivationOwner.addresses.map((a) => parse(a)[2]),
+      Number(resp.deactivationOwner.threshold),
+    );
+    const remainingBalanceOwner = PChainOwner.fromNative(
+      resp.remainingBalanceOwner.addresses.map((a) => parse(a)[2]),
+      Number(resp.remainingBalanceOwner.threshold),
+    );
+
+    return {
+      balance: BigInt(resp.balance),
+      nodeID: resp.nodeID,
+      publicKey: resp.publicKey,
+      subnetID: resp.subnetID,
+      weight: BigInt(resp.weight),
+      deactivationOwner,
+      remainingBalanceOwner,
```

Comment:
Any reason for not returning `startTime`, `minNonce` and `height`?

Reply:
Nope, simply forgot about it (I had the model in mind before I knew the response's shape).

Added in b31f3767